gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -238,6 +239,7 @@ def populate_info_dict(
     faf_pops: Dict[str, str] = FAF_POPS,
     sexes: List[str] = SEXES,
     in_silico_dict: Dict[str, Dict[str, str]] = IN_SILICO_ANNOTATIONS_INFO_DICT,
+    vrs_fields_dict: Dict[str, Dict[str, str]] = VRS_FIELDS_DICT,
```

Comment:
Question: what all is going into VRS_Fields_Dict - do we know what it will look like or is there a branch/PR where it's been added?

Reply:
ah I see it now, very helpful , thank you !